ideas/145-identity.md

```diff
@@ -0,0 +1,147 @@
+# Self-sovereign Identity
+
+## Preamble
+
+    Idea: 145-identity
+    Title: Self-sovereign Identity
+    Status: In Progress 
+    Created: 2018-05-04
+    Requires: 150-gas-abstraction 151-ens-usernames 152-friends-recovery ERC725 ERC735
```

Comment:
Considering how many dependencies this ideas has, maybe we can make the scope more minimal? So it's less about integrating in app straight away and more about finalizing PoC and influencing ERC725/735 based on desired UX. Then we can have a separate swarm for implementing it in app (and getting it audited) once requirements are done. WDYT?